extensionsGallery.json

```diff
@@ -1534,6 +1534,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "39",
+					"extensionName": "azuredatastudio-managed-instancee",
+					"displayName": "Managed Instance dashboard",
+					"shortDescription": "Display Managed Instance properties, state, and health information",
+					"publisher": {
+						"displayName": "Jovan Popovic",
+						"publisherId": "jocapc",
+						"publisherName": "jocapc"
+					},
+					"versions": [
+						{
+							"version": "0.1.1",
+							"lastUpdated": "4/30/2019",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",

```

Comment:
We currently don't support in-product installation of non-Microsoft published extensions due to policy constraints.  We're looking at various marketplace options to support this in the future, but currently external extensions need to be published with `"assetType": "Microsoft.SQLOps.DownloadPage"` and a link to a download page.

Reply:
Updated, thanks.